holmes/core/supabase_dal.py

```diff
@@ -96,8 +117,31 @@ def __load_robusta_config() -> Optional[RobustaToken]:
             for conf in config.sinks_config:
                 if "robusta_sink" in conf.keys():
                     token = conf["robusta_sink"].get("token")
-                    return RobustaToken(**json.loads(base64.b64decode(token)))
-
+                    if not token:
+                        raise Exception(
+                            f"No token provided in robusta_sink. "
+                            f"Please set a valid Robusta UI token. "
+                            f"See https://docs.robusta.dev/master/configuration/sinks/RobustaUI.html#configuring-the-robusta-ui-sink for instructions."
+                        )
+                    if "{{" in token:
+                        raise ValueError(
+                            f"The token appears to be a templating placeholder (e.g. `{{ env.UI_SINK_TOKEN }}`). "
+                            f"Ensure your Helm chart or environment variables are set correctly. "
+                            f"If you store the token in a secret, you must also pass "
+                            f"the environment variable ROBUSTA_UI_TOKEN to Holmes. "
+                            f"See https://docs.robusta.dev/master/configuration/ai-analysis.html#configuring-holmesgpt-access-to-saas-data for instructions."
```

Comment:
At some point we may add a check to make sure all URLs in our codebase are still valid. Cc @moshemorad. Not required for this PR.